ush/python/pygfs/task/aero_analysis.py

```diff
@@ -0,0 +1,238 @@
+        fms_bkg_file_template = os.path.join(comin_ges_atm, 'RESTART', f'{cdate_fv3}.fv_tracer.res.tile1.nc')
+        incvars = ['dust1', 'dust2', 'dust3', 'dust4', 'dust5',
+                   'seas1', 'seas2', 'seas3', 'seas4',
+                   'so4', 'oc1', 'oc2', 'bc1', 'bc2']
+        for itile in range(1, 7):
+            inc_path = fms_inc_file_template.replace('tile1', f'tile{itile}')
+            bkg_path = fms_bkg_file_template.replace('tile1', f'tile{itile}')
+            with nc.Dataset(inc_path, mode='r') as incfile:
+                with nc.Dataset(bkg_path, mode='a') as rstfile:
+                    for vname in incvars:
+                        increment = incfile.variables[vname][:]
+                        bkg = rstfile.variables[vname][:]
+                        anl = bkg + increment
+                        rstfile.variables[vname][:] = anl[:]
+                        try:
+                            rstfile.variables[vname].delncattr('checksum')  # remove the checksum so fv3 does not complain
+                        except AttributeError:
+                            pass  # checksum is missing, move on
+
+    def get_bkg_dict(self):
+        """
+        Return FileHandler config for model backgrounds
+        """
+        # NOTE for now this is FV3 RESTART files and just assumed to be fh006
+        comin_ges = self.config['COMIN_GES']
+        # NOTE that while 'chem' is the $componenet, the aerosol fields are with the 'atmos' tracers
+        comin_ges_atm = comin_ges.replace('chem', 'atmos')
+        rst_dir = os.path.join(comin_ges_atm, 'RESTART')  # for now, option later?
+        # date variable string format
+        cdate_fv3 = self.current_cycle.strftime('%Y%m%d.%H%M%S')
+        # get FV3 RESTART files, this will be a lot simpler when using history files
+        ntiles = 6  # global
+        # aerosol DA only needs core/tracer
+        bkglist = []
+        basename = f'{cdate_fv3}.coupler.res'
+        bkglist.append([os.path.join(rst_dir, basename), os.path.join(self.config['DATA'], 'bkg', basename)])
+        for t in range(1, ntiles + 1):
+            basename = f'{cdate_fv3}.fv_core.res.tile{t}.nc'
+            bkglist.append([os.path.join(rst_dir, basename), os.path.join(self.config['DATA'], 'bkg', basename)])
+            basename = f'{cdate_fv3}.fv_tracer.res.tile{t}.nc'
+            bkglist.append([os.path.join(rst_dir, basename), os.path.join(self.config['DATA'], 'bkg', basename)])
+        bkg_dict = {
+            'mkdir': [os.path.join(self.config['DATA'], 'bkg')],
+            'copy': bkglist,
+        }
+        return bkg_dict
+
+    def get_berror_dict(self):
+        """
+        Return FileHandler configuration for berror
+        """
+
+        ntiles = 6  # global
+        # aerosol static-B needs nicas, cor_rh, cor_rv and stddev files.
+        b_dir = self.config['BERROR_DATA_DIR']
+        berror_list = []
+        berror_list.append([
```

Comment:
This should also be handled by a YAML file if possible... I will do this as part of this PR

Reply:
actually there are a lot of loops here, instead should just use a BERROR date env var or something like that...